### DISCUSSION TITLE
Delete property?!

### DISCUSSION THREAD

**Discussion by librafrog** [AppFlowy-IO/AppFlowy#3015] [General help wanted]
I dodn't know what, the problem might be, but I have a column/property that I can't delete or edit. The only option is Hide. Any clue 🤔 
![image](https://github.com/AppFlowy-IO/AppFlowy/assets/79138819/b9540207-fbac-4270-b677-f3ec031044bd)

**Comment by richardshiue** [comment]
Hi @librafrog, thanks for raising this issue. This field that you are seeing is the title field that you will see when you open up the row as a page, or view the entries in a kanban/calendar. As such, we've deliberately made it so you cannot edit its type, duplicate or delete it. Though, I see that this information isn't immediately communicated to the user. Do you have a suggestion on how to improve this?

**Comment by librafrog** [comment]
Maybe a standart icon showing that this is indeed the title filed which we'll obviously see more and more, as our DB gets larger 😄 

- ![square-minus_icon](https://github.com/AppFlowy-IO/AppFlowy/assets/79138819/eaaaf1ee-f267-4e4a-be24-203e1734fcf8)
- ![chart_bar_icon](https://github.com/AppFlowy-IO/AppFlowy/assets/79138819/d8358c9a-6e40-450f-a571-8d6181b4b0e1)

**Reply by annieappflowy** [reply]
I love this idea!
